release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -86,7 +86,12 @@ By default, the installation program now deploys a Microsoft Azure cluster using
 
 [id="ocp-4-11-worker-nodes"]
 ==== Default AWS and VMware vSphere compute node resources
-Beginning with {product-title} 4.11, by default, the installation program now deploys AWS and VMware vSphere compute nodes with 4 vCPUs and 16 GB of virtual RAM.
+Beginning with {product-title} 4.11, by default, the installation program now deploys AWS and VMware vSphere compute nodes with 4 vCPUs and 16 GB of virtual RAM
+
+[id="ocp-4-11-heterogeneous-tech-preview"]
+==== {product-title} on Heterogeneous architectures (Technology Preview)
+{product-title} 4.11 introduces Heterogeneous architecture cluster support using Azure and AWS installer-provisioned infrastructure in Technology preview.
+This feature offers the ability to deploy a cluster with `x86_64` control planes and mixed `x86_64`/`arm64` worker nodes exclusively for Azure and AWS instances.
```

Comment:
would be good to call out that adding workers of arm64 architecture is a day2 operation

Reply:
I'll look into that, thank you!